### DISCUSSION TITLE
Comparing the performance between the Minimal API and classic Controllers

### DISCUSSION THREAD

**Discussion by giscus** [linkdotnet/Blog.Discussions#119] [General]
# Comparing the performance between the Minimal API and classic Controllers

Today, we are going to benchmark the performance of the Minimal API in ASP.NET 9 (and for reference against ASP.NET 8 as well) against the classic Controllers. We will test a few scenarios and check how the performance of the Minimal API compares to the classic Controllers. Let's get right into it!

https://steven-giesel.com/blogPost/698c45c3-58c5-4157-b4da-2cde4e27862e

**Comment by StephenCleary** [comment]
Pretty sure you mean David Fowler, not Martin Fowler! :D

**Reply by linkdotnet** [reply]
Oh boy - yes indeed! Sometimes my brain is somewhere else.

In any case - thanks for that and I did correct the name

**Comment by Britvich** [comment]
I wonder if [CreateEmptyBuilder](https://learn.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.builder.webapplication.createemptybuilder?view=aspnetcore-9.0) or [CreateSlimBuilder](https://learn.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.builder.webapplication.createslimbuilder?view=aspnetcore-9.0) would be any different?

**Reply by linkdotnet** [reply]
Interesting thought. I try to amend this!
